ChatSX/notifications/index.ts

```diff
@@ -35,20 +45,105 @@ export default async function getNotification(remoteMessage: any) {
   } else {
     decodedAndroidMsg = decodeForNotifications(message);
   }
-  await notifee.displayNotification({
-    title: remoteMessage?.data?.title,
-    body: isIOS ? decodedIOSMsg : decodedAndroidMsg,
-    data: remoteMessage?.data,
-    id: remoteMessage?.messageId,
-    android: {
-      channelId,
-      // smallIcon: remoteMessage?.community_logo, // optional, defaults to 'ic_launcher'.
-      // pressAction is needed if you want the notification to open the app when pressed
-      pressAction: {
-        id: 'default',
-        launchActivity: 'default',
-      },
-      importance: AndroidImportance.HIGH,
-    },
-  });
+
+  if (!remoteMessage?.data?.route) {
+    return;
+  }
+
+  const route = await getRoute(remoteMessage?.data?.route);
+  const routeGot = route?.params?.navigationRoute;
+
+  if (routeGot === 'collabcard' && routeGot) {
+    const UUID =
+      Credentials.userUniqueId.length > 0
+        ? Credentials.userUniqueId
+        : users?.userUniqueID;
+    const userName =
+      Credentials.username.length > 0
+        ? Credentials.username
+        : users?.username;
+
+    const payload = {
+      uuid: UUID, // uuid
+      userName: userName, // user name
+      isGuest: false,
+    };
+    const res = await myClient.initiateUser(payload);
+    if (res?.success === true) {
+      const response = await myClient?.getUnreadConversationNotification();
+      if (response?.success === false && isIOS) {
+        await notifee.displayNotification({
+          title: remoteMessage?.data?.title,
+          body: isIOS ? decodedIOSMsg : decodedAndroidMsg,
+          data: remoteMessage?.data,
+          id: remoteMessage?.messageId,
+          android: {
+            channelId,
+            // pressAction is needed if you want the notification to open the app when pressed
+            pressAction: {
+              id: 'default',
+              launchActivity: 'default',
+            },
+            importance: AndroidImportance.HIGH,
+          },
+        });
+      } else {
```

Comment:
what is the need to call, initiate api, if user in on iOS, these changes should be for android only.